matplotlib-nogui/disable_plugins.patch

```diff
@@ -0,0 +1,39 @@
+--- setupext.py
++++ setupext.py
+@@ -1050,22 +1050,8 @@
+     name = "qhull"
+ 
+     def check(self):
+-        self.__class__.found_external = True
```

Comment:
There are several reasons:

1. there is no conda package for qhull (or I couldn't find it)
1. since matplotlib has a local copy and it seems to work fine, we had better stick to it explicitly

P.S. Unfortunately, I don't recall at the top of my head what was the exact issue.